### DISCUSSION TITLE
[Script request] qBit_Manage

### DISCUSSION THREAD

**Discussion by oOStroudyOo** [community-scripts/ProxmoxVE#977] [Request script]
### Application Name

qBit_Manage

### Website

https://github.com/StuffAnThings/qbit_manage

### Description

This tool will help manage tedious tasks in qBittorrent and automate them. Tag, categorize, remove Orphaned data, remove unregistered torrents and much much more.

It would be good to have the ability to add this into it's own LXC, as well as an optional install along side qBittorrent.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by clipz98** [comment]
Bringing this from the dead. Seems like it'd be an easy addon script for the existing qbittorrent script OR (preferably) a standalone LXC. I'm about to get into this and create my own LXC manually, but checked here first to see if anyone already requested.

Please consider this older request team! Here's the installation wiki for reference too: https://github.com/StuffAnThings/qbit_manage/wiki

**Comment by Pr0mises** [comment]
It got easier as they provide standalone binaries as of now:
https://github.com/StuffAnThings/qbit_manage/wiki/Installation#standalone-binary-installation

**Comment by MickLesk** [comment]
Is this an Plugin / Addon for existing qbittorrent or really an separate LXC?

**Reply by oOStroudyOo** [reply]
Either is fine.
It only communicates with qBittorrent as far as I can tell.

It does now come with a WebUI so if packaged with qBittorent, the final message will need to display the qbittorrent and qbit_manage IP:Port so the user knows where to go for setup.

**Comment by Pr0mises** [comment]
I'm running it in a separate LXC but having it in the same lxc could be fine? It only loads from a config file and all requests go to the qbit instance

**Comment by clipz98** [comment]
I guess it can be kinda both, but it's kind of its own beast that should probably be a separate LXC from qBittorrent itself.
